x-pack/plugins/security_solution/public/management/components/endpoint_responder/endpoint_response_actions_console_commands.tsx

```diff
@@ -62,21 +92,58 @@ const COMMENT_ARG_ABOUT = i18n.translate(
   { defaultMessage: 'A comment to go along with the action' }
 );
 
-export const getEndpointResponseActionsConsoleCommands = (
-  endpointAgentId: string
-): CommandDefinition[] => {
+const DISABLED_COMMAND_INFO = i18n.translate(
+  'xpack.securitySolution.endpointConsoleCommands.suspendProcess.disabledCommandInfo',
+  { defaultMessage: 'This endpoint does not support this commmand' }
+);
+
+const DISABLED_COMMAND = i18n.translate(
+  'xpack.securitySolution.endpointConsoleCommands.suspendProcess.disabledCommand',
+  { defaultMessage: 'Disabled' }
```

Comment:
We should probably use different terminology here.  These commands are unsupported by the current Endpoint, so we should say "Unsupported".